Cargo.toml

```diff
@@ -114,6 +114,7 @@ members = [
 	"frame/proxy",
 	"frame/nomination-pools",
 	"frame/nomination-pools/benchmarking",
+	"frame/nomination-pools/test-staking",
```

Comment:
This is a new pallet with end to end tests around staking and pools. I can recommend giving it a read. Other test changes are not really important.